static/src/javascripts/bootstraps/enhanced/accessibility.js

```diff
@@ -82,13 +84,13 @@ define([
 
         React.render(
             React.createElement(Accessibility),
-            document.getElementById(module.DOM_ID),
+            document.getElementById(DOM_ID),
```

Comment:
We could define `module` at the top, along with `module.DOM_ID`, and then add `init` to the `module` object afterwards before exporting. I deliberately wanted to avoid this mutation though.

I've seen other modules use this technique as a hacky means of testing. Perhaps those module should be using Squire to do proper dependency injection instead.